modules/ROOT/pages/metrics/coding.adoc

```diff
@@ -0,0 +1,16 @@
+include::{partialsdir}/attributes.adoc[]
+
+include::configuring-dev-env-metrics.adoc[]
+
+== Setting up {metrics-service} service SDK
```

Comment:
we don't use this heading elsewhere

Reply:
dang you and your 'facts'... I've created https://issues.jboss.org/browse/AEROGEAR-7893 address this later ;)